realm/main.cc

```diff
@@ -810,7 +818,7 @@ void shard_task(const void *args, size_t arglen, const void *userdata,
                       .at(point - first_point)
                       .at(dep)
                       .at(slot),
-                    fid, graph.output_bytes_per_task,
+                    fid, sizeof(char),
```

Comment:
Maybe `sizeof(int8_t)`? I'm sure `char` is 1 byte on almost every system... but when it isn't, we're going to be off.

(Same below.)

Reply:
I use char because we use sizeof(char) when creating fields. https://github.com/StanfordLegion/task-bench/blob/fix_realm/realm/main.cc#L344
I can change both of them to int8_t.

Reply:
You're right. Keep it.